deployments/bridges/westend-millau/entrypoints/relay-headers-westend-to-millau-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -xeu
+
+sleep 3
+curl -v http://millau-node-alice:9933/health
```

Comment:
```suggestion
curl -v http://millau-node-alice:9933/health
curl -v https://westend-rpc.polkadot.io/health
```

Reply:
This command leaves errors in logs. So even though it returns health of the node, there are some lines with "error: " inside

Reply:
Can you elaborate on the errors?  It would be good to quick feedback from `curl` if `westend-rpc.polkadot.io` cannot be connected to before even running the `init-bridge` command.

Reply:
```
+ curl -v https://westend-rpc.polkadot.io/health
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 104.21.60.44:443...
* TCP_NODELAY set
* Connected to westend-rpc.polkadot.io (104.21.60.44) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* error setting certificate verify locations:
  CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
* Closing connection 0
curl: (77) error setting certificate verify locations:
  CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
{"isSyncing":false,"peers":4,"shouldHavePeers":true}
```

Reply:
Seems there is no CA cerificates? [Similar issue here](https://github.com/curl/curl/issues/6090). Perahps we should either:
1. `update-ca-certificates` in the docker image (or install some missing package)
2. Disable SSL checking (`--insecure`?) cause without CA certificates it doesn't really work anyway I guess.

Reply:
I don't mind dealing with it in a separate issue - it isn't about Westend -> Millau bridge.